packages/ag-charts-community/src/chart/series/polar/pieSeries.ts

```diff
@@ -334,7 +340,14 @@ export class PieSeries extends PolarSeries<PieNodeDatum, Sector> {
         this.calloutLabelSelection = Selection.select(pieCalloutLabels, Group);
         this.sectorLabelSelection = Selection.select(pieSectorLabels, Text);
         this.innerLabelsSelection = Selection.select(innerLabels, Text);
-        this.innerCircleSelection = Selection.select(this.innertCircleGroup, Circle);
+        this.innerCircleSelection = Selection.select(this.innerCircleGroup, Circle);
+
+        for (const circle of [this.zerosumInnerRing, this.zerosumOuterRing]) {
+            circle.fillOpacity = 0;
+            circle.stroke = 'black';
+            circle.strokeWidth = 1;
+            circle.strokeOpacity = 1;
```

Comment:
🤔 Should these relate to the pie series options? I.e. if I set a `series[].strokeWidth` should it be applied here?